docs/source/tutorials/ram_usage.rst

```diff
@@ -0,0 +1,54 @@
+RAM usage
+=========
+
+All devices have 0.5GB (4Gbit) on-board RAM, which is used for firmware (about 15MB), assets (a few KB up to 100MB+, eg. NN models), and other
```

Comment:
All devices have 512MiB
^ IMO - as we use megabytes everywhere else, easier to calculate in head